l1-contracts/src/core/sequencer_selection/Leonidas.sol

```diff
@@ -9,7 +9,6 @@ import {Ownable} from "@oz/access/Ownable.sol";
 import {SignatureLib} from "./SignatureLib.sol";
 import {SampleLib} from "./SampleLib.sol";
 import {Constants} from "../libraries/ConstantsGen.sol";
-import {MessageHashUtils} from "@oz/utils/cryptography/MessageHashUtils.sol";
```

Comment:
Should we still have some prefix to avoid people being able to use it directly for an eth transaction? This is part of why there is starting `\x19` of EIP712 https://github.com/0xProject/EIPs/blob/master/EIPS/eip-712.md